src/XmlApiTestCase.php

```diff
@@ -60,6 +60,23 @@ private function assertXmlHeader(Response $response)
      */
     private function assertXmlResponseContent(Response $response, $filename)
     {
-        parent::assertResponseContent($response, $filename, 'xml');
+        $actualResponse = $response->getContent();
```

Comment:
``actual`` makes sense here because we compare 2 contents, actual and expected. If we want to be more explicit, let's say ``actualResponseContent``.